src/app/tictactoe/TestTicTacToe.java

```diff
@@ -11,16 +11,24 @@
     public static InputManager input;
 
     public TestTicTacToe() {
-        display = new Display(600, 600, "TicTacToe");
-        input = new InputManager();
-        map = new MovingAndMap();
         runGame();
     }
 
     public void runGame() {
 
+        display = new Display(600, 600, "TicTacToe");
+        input = new InputManager();
+        map = new MovingAndMap();
```

Comment:
These three lines of code are repeated in a different place as well. Maybe refactor into a method.